starter-code/views/gallery.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="../public/css/style.css">
```

Comment:
Como o css está na pasta public, você não precisa referenciar ele.
```suggestion
  <link rel="stylesheet" href="css/style.css">
```